examples/aws-ses/package.json

```diff
@@ -4,23 +4,24 @@
   "license": "MIT",
   "private": true,
   "sideEffects": false,
+  "type": "module",
   "main": "./dist/index.js",
-  "module": "./dist/index.mjs",
-  "types": "./dist/index.d.ts",
   "files": [
     "dist/**"
   ],
   "scripts": {
-    "build": "tsup src/index.tsx --format esm,cjs --dts --external react",
-    "dev": "tsup src/index.tsx --format esm,cjs --dts --external react --watch",
+    "build": "tsup-node src/index.tsx --format esm --target node18",
```

Comment:
should we try node20? as it's the current LTS

Reply:
Agreed, changing that